package.json

```diff
@@ -87,7 +87,6 @@
     "typescript": "^4.1.2"
   },
   "dependencies": {
-    "@alma/client": "^1.1.1",
```

Comment:
I founded a reference into package.lock. I think  it can be remove to. Can you try ?

`"@alma/client": "^1.1.1",`